aip/general/0148.md

```diff
@@ -137,6 +128,25 @@ used by server-side policies, such as IAM conditions. Annotations exist to
 allow client tools to store their own state information without requiring a
 database.
 
+### Well known string fields
+
+#### IP address
+
+A field that represents an IP address **must** comply with the following:
+
+* use type `string`
+* use the name `ip_address` or end with the suffix `_ip_address` e.g.
+  `resolved_ip_address`
+* specify the IP address version format via one of the supported formats `IPV4`,
+  `IPV6`, or if it can be either, `IPV4_OR_IPV6` (see [AIP-202][aip-202])
+
+#### uid
+
+The output only `string uid` field refers to a system-assigned, unique
```

Comment:
I'd personally remove the comma from here, but I'm not going to block on that of course :)